Jenkinsfile

```diff
@@ -106,15 +106,17 @@ node('ubuntu-zion') {
     stage('Evaluate') {
       
       //Create tar of our image
-      OsTools.runSafe(this, "docker save ${imageName} -o ${env.WORKSPACE}/${tarName}")
+      dir('build') {
+        OsTools.runSafe(this, "docker save ${imageName} -o ${tarName}")
       
-      //decide which stage we are creating
-      def theStage = branch == 'master' ? (params.push_image ? 'release' : 'stage-release') : 'build'
+        //decide which stage we are creating
+        def theStage = branch == 'master' ? (params.push_image ? 'release' : 'stage-release') : 'build'
 
-      //run the evaluation
-      nexusPolicyEvaluation iqStage: theStage, iqApplication: iqApplicationId,
-        iqScanPatterns: [[scanPattern: '*.tar']],
-        failBuildOnNetworkError: true
+        //run the evaluation
+        nexusPolicyEvaluation iqStage: theStage, iqApplication: iqApplicationId,
+          iqScanPatterns: [[scanPattern: '*.tar']],
```

Comment:
Not sure but I suspect this might need changing to `build/*.tar` now. At least the old Hudson integration resolved patterns relative to the workspace root, not whatever the current directory is.

Reply:
> Not sure but I suspect this might need changing to `build/*.tar` now. At least the old Hudson integration resolved patterns relative to the workspace root, not whatever the current directory is.

Worth a try https://github.com/sonatype/docker-nexus-iq-server/pull/44/commits/7d52d21b73a7c1a010e75d8352500b57e3a01cee